src/sass/_instance_list.scss

```diff
@@ -213,3 +205,11 @@
     }
   }
 }
+
+.table-column-select-list {
```

Comment:
Why move it out of the instance list selector?

Reply:
Try moving it back in and you'll get this:

![image](https://user-images.githubusercontent.com/56583786/233035875-7ec48022-b238-4851-8efd-0e84ebb52d84.png)